### PR TITLE
Set up PostgreSQL database and connection string

### PR DESCRIPTION
**Pull request by WesleyHarris**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,5 +46,8 @@ export default tseslint.config({
     ...react.configs.recommended.rules,
     ...react.configs['jsx-runtime'].rules,
   },
-})
+});
 ```
+
+//
+// Hi, Matt!
```
Comment (matthewjenner) [inline README.md:53]: No....

**Review by matthewjenner — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (WesleyHarris) [reply README.md:53]: Why not?

**Comment by matthewjenner** [comment]
Looks like you are missing a .gitignore file, so you are checking in secrets (.env file) as well as binaries (obj/ and bin/ folders)

Ask Claude to generate you a .gitignore file for a react project with a .net core api backend and how to fix your branch on the remote if you checked in some files that should have been caught by the ignore file.

**Comment by WesleyHarris** [comment]
Thanks, bro. Appreciate you looking at that. I knew that shouldn't have been in there but skipped a step when I got trapped in pgAdmin hell.

**Review by matthewjenner — APPROVED** [review]

**Pull request merged**